From 1dc88c054670cdef6d326201f3e8f5f66fd6aa86 Mon Sep 17 00:00:00 2001
From: Johannes Schindelin <johannes.schindelin@gmx.de>
Date: Fri, 2 Jun 2017 17:11:09 +0200
Subject: [PATCH 4/6] RunGrepTest: skip test with absolute Unix path

In MSYS2's context, the `/` path is converted to a Windows path, hence
the expectations of the test script were not met. Simply ignore this
test case.

Signed-off-by: Johannes Schindelin <johannes.schindelin@gmx.de>
---
 RunGrepTest         |  6 +++---
 testdata/grepoutput | 17 -----------------
 2 files changed, 3 insertions(+), 20 deletions(-)

diff --git a/RunGrepTest b/RunGrepTest
index 823f9e9..7b93f95 100755
--- a/RunGrepTest
+++ b/RunGrepTest
@@ -570,9 +570,9 @@ echo "---------------------------- Test 109 -----------------------------" >>tes
 (cd $srcdir; $valgrind $vjs $pcre2grep -cq lazy ./testdata/grepinput*) >>testtrygrep
 echo "RC=$?" >>testtrygrep
 
-echo "---------------------------- Test 110 -----------------------------" >>testtrygrep
-(cd $srcdir; $valgrind $vjs $pcre2grep --om-separator / -Mo0 -o1 -o2 'match (\d+):\n (.)\n' testdata/grepinput) >>testtrygrep
-echo "RC=$?" >>testtrygrep
+#echo "---------------------------- Test 110 -----------------------------" >>testtrygrep
+#(cd $srcdir; $valgrind $vjs $pcre2grep --om-separator / -Mo0 -o1 -o2 'match (\d+):\n (.)\n' testdata/grepinput) >>testtrygrep
+#echo "RC=$?" >>testtrygrep
 
 echo "---------------------------- Test 111 -----------------------------" >>testtrygrep
 (cd $srcdir; $valgrind $vjs $pcre2grep --line-offsets -M 'match (\d+):\n (.)\n' testdata/grepinput) >>testtrygrep
diff --git a/testdata/grepoutput b/testdata/grepoutput
index 5f3b97c..47eda34 100644
--- a/testdata/grepoutput
+++ b/testdata/grepoutput
@@ -790,23 +790,6 @@ RC=0
 RC=0
 ---------------------------- Test 109 -----------------------------
 RC=0
----------------------------- Test 110 -----------------------------
-match 1:
- a
-/1/a
-match 2:
- b
-/2/b
-match 3:
- c
-/3/c
-match 4:
- d
-/4/d
-match 5:
- e
-/5/e
-RC=0
 ---------------------------- Test 111 -----------------------------
 607:0,12
 609:0,12
-- 
2.30.0.windows.2

